tests/testthat/test-accuracy-published.R

```diff
@@ -0,0 +1,245 @@
+
+
+test_that(".OneParExpNormalPrior works as expected", {
+
+
+  mcmc_options <- crmPack::McmcOptions(
+    burnin = 50000, step = 2,
+    samples = 1000000
+  )
+
+
+  ## One-parameter model
+  ## (A) Posterior summaries (original skeleton)
+  empty_data <- crmPack::Data(dose_grid = c(
+    1, 2.5, 5, 10, 15, 20, 25, 30,
+    40, 50, 75, 100, 150, 200, 250
+  ))
+
+  data_obs_a <- crmPack::Data(
+    x = c(
+      rep(1, 3), rep(2.5, 4), rep(5, 5),
+      rep(10, 4), rep(25, 2)
+    ),
+    y = c(
+      rep(0, 3), rep(0, 4), rep(0, 5),
+      rep(0, 4), rep(1, 2)
+    ),
+    cohort = c(
+      rep(1, 3), rep(2, 4), rep(3, 5),
+      rep(4, 4), rep(7, 2)
+    ),
+    doseGrid = c(
+      1, 2.5, 5, 10, 15, 20, 25, 30,
+      40, 50, 75, 100, 150, 200, 250
+    ),
+    ID = 1:18
+  )
+
+  model_power_a <- crmPack::OneParExpNormalPrior(
+    skel_probs = c(
+      0.01, 0.015, 0.020, 0.025, 0.03,
+      0.04, 0.05, 0.10, 0.17, 0.30,
+      0.45, 0.70, 0.80, 0.90, 0.95
+    ),
+    dose_grid = c(
+      1, 2.5, 5, 10, 15, 20, 25, 30,
+      40, 50, 75, 100, 150, 200, 250
+    ),
+    sigma2 = 1.34^2
+  )
+
+  prior_samples <- crmPack::mcmc(
+    data = empty_data,
+    model = model_power_a,
+    options = mcmc_options
+  )
+
+  ## NCRM rule with loss function
+  ncrm_loss <- crmPack::NextBestNCRMLoss(
+    target = c(0.2, 0.35),
+    overdose = c(0.35, 0.6),
+    unacceptable = c(0.6, 1),
+    max_overdose_prob = 0.25,
+    losses = c(1, 0, 1, 2)
+  )
+
+  increments_no <- crmPack::IncrementsRelative(
+    intervals = c(0, 250),
+    increments = c(2, 2)
+  )
+
+  post_samples_a <- crmPack::mcmc(data_obs_a, model_power_a, mcmc_options)
+
+  dose_rec_loss_a <- crmPack::nextBest(ncrm_loss,
```

Comment:
please run `styler` Add-in on the whole file to improve code style and indentation

Reply:
done